packages/engine/tests/portal/portal.test.ts

```diff
@@ -0,0 +1,55 @@
+import assert from 'assert'
+import { unloadScene } from '../../src/ecs/functions/EngineFunctions'
+import { initializeEngine } from '../../src/initializeEngine'
+import { engineTestSetup } from '../util/setupEngine'
+import sceneJson from '@xrengine/projects/default-project/default.scene.json'
+import { WorldScene } from '../../src/scene/functions/SceneLoading'
+import { useWorld } from '../../src/ecs/functions/SystemHooks'
+import { Engine } from '../../src/ecs/classes/Engine'
+import { EngineEvents } from '../../src/ecs/classes/EngineEvents'
+import { parseSceneDataCacheURLs } from '@xrengine/server-core/src/world/scene/scene-parser'
+
+import appRootPath from 'app-root-path'
+import dotenv from 'dotenv-flow'
+
+dotenv.config({
+  path: appRootPath.path,
+  silent: true
+})
+
+const sceneData = parseSceneDataCacheURLs(sceneJson, process.env.LOCAL_STORAGE_PROVIDER!)
+
+describe.skip('Portal', () => {
```

Comment:
skip intentional?

Reply:
yeah, it's breaking because PhysX is persisting between tests. gotta figure out a better solution to global values with mocha